<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NbNote</class>
 <widget class="QWidget" name="NbNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 211, 239);</string>
  </property>
  <widget class="QPushButton" name="pushButtonNoir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/num1.png</normaloff>:/images/num1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeuxCroche">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/num2.png</normaloff>:/images/num2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTroisCroche">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/num3.png</normaloff>:/images/num3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuatreDCroche">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/num4.png</normaloff>:/images/num4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCinqDCroche">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/num5.png</normaloff>:/images/num5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPasSub">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Pas de subdivision</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonValider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/valide.png</normaloff>:/images/valide.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelNoir">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>41</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Images/images/NoteNoirModifier.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelDeuxCroche">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Images/images/NoteCrocheModifier.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelTroisCroche">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Images/images/NoteCrocheModifier.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelQuatreDCroche">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Images/images/NoteDoubleCrocheModifier.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelCinqDCroche">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/Images/images/NoteDoubleCrocheModifier.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonValider</sender>
   <signal>clicked()</signal>
   <receiver>NbNote</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
